extensions/mentions/migrations/2022_10_21_000000_create_post_mentions_groups_table.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+use Flarum\Database\Migration;
+use Illuminate\Database\Schema\Blueprint;
+
+return Migration::createTable(
+    'post_mentions_group',
+    function (Blueprint $table) {
+        $table->integer('post_id')->unsigned();
+        $table->integer('mentions_group_id')->unsigned();
+        $table->timestamp('created_at')->nullable();
+        $table->primary(['post_id', 'mentions_group_id']);
+
+        $table->foreign('post_id')->references('id')->on('posts')->onDelete('cascade');
+        $table->foreign('mentions_group_id')->references('id')->on('groups')->onDelete('cascade');
+    }
+);
```

Comment:
> After this change, the tests fail on MySQL 5.7 - I think to do with the comment on dbal in the deleted migration.
>
> @SychO9 - https://github.com/flarum/framework/actions/runs/3311921966/jobs/5467972384
>
> https://github.com/flarum/framework/commit/9347665baa1820ed0875c6b2dd22b14fbc2768e7

Ah yeah, In that case
```suggestion
use Illuminate\Database\Schema\Blueprint;
use Illuminate\Database\Schema\Builder;

return [
    'up' => function (Builder $schema) {
        $schema->create('post_mentions_groups', function (Blueprint $table) {
            $table->integer('post_id')->unsigned();
            $table->integer('mentions_group_id')->unsigned();
            $table->timestamp('created_at')->nullable();
            $table->primary(['post_id', 'mentions_group_id']);

            $table->foreign('post_id')->references('id')->on('posts')->onDelete('cascade');
            $table->foreign('mentions_group_id')->references('id')->on('groups')->onDelete('cascade');
        });

        // do this manually because dbal doesn't recognize timestamp columns
        $connection = $schema->getConnection();
        $prefix = $connection->getTablePrefix();
        $connection->statement("ALTER TABLE `${prefix}post_mentions_group` MODIFY created_at TIMESTAMP NULL DEFAULT CURRENT_TIMESTAMP");
    },

    'down' => function (Builder $schema) {
        $schema->drop('post_mentions_groups');
    }
];
```